examples/with-magic/README.md

```diff
@@ -10,9 +10,9 @@ The login cookie is `httpOnly`, meaning it can only be accessed by the API, and
 
 ## Deploy your own
 
-Deploy the example using [Vercel Now](https://vercel.com/docs/now-cli#commands/overview/basic-usage):
+Once you have access to [the environment variables you'll need](#configuration), deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example):
```

Comment:
I'd say either remove the Vercel link, or make it the same link as the deploy button link 👍

Reply:
This is how we currently have it for every example 🤔